learn/features/xchain-plans.md

```diff
@@ -1,33 +1,67 @@
 ---
 
-# Cross-Chain Integration Plans
+# Cross-Chain Communication Methods
 
-One of the key planned features for Moonbeam is to provide an easy way for developers to use smart contracts to integrate with other chains in the Polkadot ecosystem.
+Moonbeam makes it easy for developers to build smart contracts that connect across chains, both within the Polkadot ecosystem and outside the Polkadot ecosystem. This guide will provide a fundamental overview of the underlying protocols that enable cross-chain communication and how you can leverage them to build connected contracts. For step-by-step guides of how to put these principles into practice, be sure to check out the [Interoperability Tutorials](/tutorials/interoperability/){target=\_blank}. 
 
-Polkadot defines a low-level integration protocol, called cross-chain message passing (XCMP), that can be used to facilitate communication between parachains within the Polkadot network and share trusted logic between chains on the Polkadot network, called shared protected runtime execution enclaves (SPREE). Parity is in the process of implementing XCMP and is in the design phase of SPREE as of writing this document (July 2020). Post-Polkadot mainnet launch, XCMP and SPREE support will be released as upgrades to the Polkadot relay chain. We plan to implement and support integration scenarios based on these protocols when they are available.
+Two key terms that will come up frequently in this guide are XCM and GMP. XCM refers to cross-consensus messaging, and it's Polkadot's native interoperability language that facilitates communication between Polkadot blockchains. 
 
-## An Analogy: The "Bash of Polkadot" {: #an-analogy-the-bash-of-polkadot } 
+[GMP](https://moonbeam.network/blog/what-is-gmp/){target=\_blank}, or general message passing, refers to the sending and receiving of messages within decentralized blockchain networks. While XCM *is a* type of general message passing, GMP is used colloquially to refer to cross-chain communication between Moonbeam and blockchains outside of Polkadot. Similarly, in this guide, XCM refers to cross-chain messaging within Polkadot, and GMP refers to cross-chain messaging between Moonbeam and other ecosystems outside of Polkadot. 
 
-For us, Polkadot will serve a similar function to something like Linux. Both are developer-oriented platforms that come with libraries to make building applications easier. 
 
-Recall the old Unix philosophy, where users build tools that do one job and do it well. This is similar to the specialization that we expect to happen to parachains on Polkadot. On Linux, you can combine and compose these purpose-built tools together to achieve higher order effects using a shell-like bash. We anticipate Moonbeam-based smart contracts will provide an analogous “bash-like” environment, where specialized smart contracts and parachain functionality can be composed to achieve higher order goals.
+## XCM Overview {: #xcm-overview }  
```

Comment:
I'm struggling with this section because I feel like it just duplicates content that we already have on the XCM Overview page.

I do really like the "Happy Path of a Cross-Chain Message" section under GMP, and so I'm thinking that we should try to keep the XCM and GMP sections like apples to apples and have the same sort of section under XCM. Maybe because this content is structured pretty closely to how it is on the overview page, that's where my problem lies, and if we can make it more casual here, and layout the flow of a cross-chain transaction via VMP and then also via HRMP in a similar manner that might be beneficial.

I don't know 🤷‍♀️  -- what do you think?

Reply:
I simplified it a bit to get rid of the XCM overview section and shuffled some things around, also putting the quick reference table in above.

Reply:
looks great!